themes/src/main/resources/theme/base/admin/resources/partials/realm-tokens.html

```diff
@@ -121,7 +121,7 @@
                     <option value="Days">{{:: 'days' | translate}}</option>
                 </select>
             </div>
-            <kc-tooltip>{{:: 'login-timeout' | translate}}</kc-tooltip>
```

Comment:
I would ask you to create a separated JIRA for this one. The reason is that those changes are beyond the scope of KEYCLOAK-3446 (pt_BR localization). It is really an issue.